ep/build-velox/src/setup-rhel.sh

```diff
@@ -0,0 +1,393 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
```suggestion
```

Is this copied from any Facebook repos? Add ep/build-velox/src/setup-rhel.sh to the NOTICE file to declare the authority, otherwise remove it